third_party/amd/lib/TritonAMDGPUTransforms/OptimizeEpilogue.cpp

```diff
@@ -32,8 +32,11 @@
 #include "triton/Dialect/TritonGPU/IR/Dialect.h"
 #include "triton/Dialect/TritonGPU/Transforms/Passes.h"
 #include "triton/Dialect/TritonGPU/Transforms/Utility.h"
+#include "triton/Tools/LayoutUtils.h"
 
 using namespace mlir;
+using namespace mlir::triton;
```

Comment:
Nit: Let's not just blanketly use all symbols in these namespaces. Prefer qualifying the usage if just once, or `using` specific symbols if multiple times.